src/engine.ts

```diff
@@ -131,6 +133,8 @@ export class Engine<A extends string = never, T extends Task<A> = Task<A>> {
       task.do instanceof Location ? task.do : undefined
     );
     macro.save();
+    writeCcs("[ default ]\nscrollwhendone", "scrollwhendone");
+    cliExecute("ccs scrollwhendone");
```

Comment:
Do we need to write this ccs and set it every time? Maybe just create it once in the constructor?